translations/en-us.yml

```diff
@@ -1610,6 +1610,8 @@ osf-components:
             error: 'Error updating contributor bibliographic status.'
         reorderContributor:
             success: 'Contributor order updated.'
+        noAffiliations: 'No institutional affiliations to show'
```

Comment:
isn't this dropdown supposed to show users (profile info) education and employment? If so, better to have `No education history to show` and `No employment history to show`. This is confusing as is in that users institutional affiliations may have nothing to do with their employment/education.